### PR TITLE
RavenDB-14965 Time-Series indexing progress - showing items/sec or do…

### PR DESCRIPTION
**Pull request by Danielle9897**
…cs/sec

### CODE REVIEW

File: src/Raven.Client/Documents/Indexes/IndexProgress.cs [hunk 1]
```diff
@@ -8,6 +8,8 @@ public class IndexProgress
 
         public IndexType Type { get; set; }
 
+        public IndexSourceType SourceType { get; set; }
```
Comment (ml054) [inline src/Raven.Client/Documents/Indexes/IndexProgress.cs:11]: like I said I think we don't want to expose that as this is constant property - and should be part of index output not progress. Even if we have race and index type isn't send we simply don't show such item as we don't know about this index. we can have a look at details tomorrow.

**Review by ml054 — CHANGES_REQUESTED** [review]

File: src/Raven.Client/Documents/Indexes/IndexProgress.cs (hunk 1 above)
Comment (ayende) [reply src/Raven.Client/Documents/Indexes/IndexProgress.cs:11]: What about multi map indexes that have both time series / counters / docs?

Comment (ml054) [reply src/Raven.Client/Documents/Indexes/IndexProgress.cs:11]: wow, I wasn't even aware we can mix index sources. Then I assume we have to introduce new type of index source: Mixed?

Comment (ppekrol) [reply src/Raven.Client/Documents/Indexes/IndexProgress.cs:11]: At this point we do not support mixing the sources of maps. I will create an issue to throw when something like that happens, till we want to actually support that.

Comment (ayende) [reply src/Raven.Client/Documents/Indexes/IndexProgress.cs:11]: We do need to take this into account, because this is very likely going to _be_ a feature.

Comment (arekpalinski) [reply src/Raven.Client/Documents/Indexes/IndexProgress.cs:11]: I don't see the problem with adding `SourceType ` here, we already have `public IndexType Type { get; set; }`

Comment (arekpalinski) [reply src/Raven.Client/Documents/Indexes/IndexProgress.cs:11]: For now let's stay with current implementation here - introducing the mixed sources will require much more changes in the studio anyway

**Review by ml054 — APPROVED** [review]

**Pull request merged**